src/utils/dev.ts

```diff
@@ -157,21 +157,22 @@ export const getSiteInformation = async ({ api, offline, site, siteInfo }) => {
   }
 }
 
-// @ts-expect-error TS(7006) FIXME: Parameter 'source' implicitly has an 'any' type.
-const getEnvSourceName = (source) => {
-  // @ts-expect-error TS(7053) FIXME: Element implicitly has an 'any' type because expre... Remove this comment to see the full error message
-  const { name = source, printFn = chalk.green } = ENV_VAR_SOURCES[source] || {}
+const getEnvSourceName = (source: string) => {
+  const { name = source, printFn = chalk.green } = ENV_VAR_SOURCES[source] ?? {}
```

Comment:
lots of type errors here. I think an enum or a union of literal strings is needed.